src/vs/workbench/services/runtimeStartup/common/runtimeStartup.ts

```diff
@@ -320,7 +321,9 @@ export class RuntimeStartupService extends Disposable implements IRuntimeStartup
 				// storage, but it is possible that this workspace will be
 				// re-opened without an interleaving quit (e.g. if multiple
 				// Positron windows are open).
-				e.veto(this.clearWorkspaceSessions(), 'positron.runtimeStartup.clearWorkspaceSessions');
+				if (!isWeb) {
+					e.veto(this.clearWorkspaceSessions(), 'positron.runtimeStartup.clearWorkspaceSessions');
+				}
```

Comment:
One more thought! Maybe a comment here noting that async operations are not supported in Web, and that's why we only veto on Desktop

Reply:
Oh that's a good idea! https://github.com/posit-dev/positron/pull/6091/commits/c4d22dd8baaf894375460856076c7de18cf23ad4